projects/smart-table/src/lib/smart-table/smart-table.component.html

```diff
@@ -27,15 +25,16 @@
     </div>
     <div class="aui-smart-table-filter a-input" *ngIf="totalResults > 0">
       <div class="a-input__wrapper">
-        <button class="a-button-outline a-button-export has-icon-left" (click)="exportToExcel()">
-          <span class="fa fa-file-excel-o"></span> <span class="a-export-label">{{ options.translations?.export || 'Export'}}</span>
+        <button class="a-button-outline a-button-export has-icon-left" (click)="exportToExcel()" [attr.aria-label]="options.translations?.export || 'Export'">
```

Comment:
```suggestion
        <button class="a-button-outline a-button-export has-icon-left" (click)="exportToExcel()" [attr.aria-label]="options.translations?.export || 'Exporteer'">
```
Wij gebruiken standaard altijd NL.